lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -275,17 +287,17 @@ private JWEObject signEncryptJar(
             String oauthState,
             String govukSigninJourneyId,
             String criId)
-            throws HttpResponseExceptionWithErrorBody, ParseException, JOSEException {
+            throws HttpResponseExceptionWithErrorBody, ParseException, JOSEException,
+                    UnknownEvidenceTypeException {
+
+        List<SignedJWT> credentials = getSignedJWTs(userId);
+
         SharedClaimsResponse sharedClaimsResponse =
-                getSharedAttributesForUser(ipvSessionItem, userId, criId);
+                getSharedAttributesForUser(ipvSessionItem, credentials, criId);
```

Comment:
This will definitely conflict with Rohit's PR - you probably need to rebase after he merges